documentation/shared/attributes.adoc

```diff
@@ -42,14 +42,22 @@
 :kafka-exporter-project: link:https://github.com/danielqsj/kafka_exporter[Kafka Exporter^]
 
 //OAuth attributes and links
+:strimzi-project-name: Strimzi
+:keycloak-project-name: Keycloak
```

Comment:
Do not use variables for this. Just write the names in the text.

Reply:
I suppose any alternative mechanism can't really be a hundred percent reliable and can result in occasional replacing in some place where you don't want to replace it?

Reply:
In the past we decided to not use any variables for project names etc. It is unreliable and just makes writting the docs harder.